cpp/community_detection_module/community_detection_module.cpp

```diff
@@ -6,46 +6,86 @@
 namespace {
 
 const char *kProcedureGet = "get";
+const char *kProcedureGetSubgraph = "get_subgraph";
 
 const char *kFieldNode = "node";
 const char *kFieldCommunity = "community_id";
 
-const char *kArgumentWeightProperty = "weight";
+// const char *kArgumentWeightProperty = "weight";
```

Comment:
Why is this not used?

Reply:
Good catch! I commented it out because of it being unused, but now I noticed it has mistakenly been left out from line 71 in the main branch code.